build-a-game-playing-agent/agent_test.py

```diff
@@ -125,8 +122,7 @@ def score(self, game, player):
 
 
 def makeEvalStop(limit, timer, value=None):
-
-    class EvalStop():
+    class EvalStop:
```

Comment:
In Python 2.x it always need to be 'class ClassName(object): ...` or you can get strange behavior.  
In Python 3.x the preferred declaration syntax is `class ClassName:` unless you are specifically metaclassing or inheriting (http://docs.python.org/tutorial/classes.html#class-definition-syntax)